doc/programs.md

```diff
@@ -39,6 +39,7 @@
 - COMIFY, convert HEX to COM format.
 - COMPLR, Lisp compiler.
 - COMSAT, mail server.
+- COOKIE BEAR, prank.
```

Comment:
Something like "- CHECK, Cookie Bear prank." might be clearer, although I appreciate clarity may be missing the point...